build-logic/src/main/kotlin/authmgr-jreleaser.gradle.kts

```
@@ -0,0 +1,175 @@
+
+  project {
+    name.set("Dremio Iceberg AuthManager")
+    description.set("Dremio AuthManager for Apache Iceberg")
+    authors.set(listOf("Dremio"))
+    license.set("Apache-2.0")
+    links {
+      homepage.set("https://github.com/dremio/iceberg-auth-manager")
+      bugTracker = "https://github.com/dremio/iceberg-auth-manager/issues"
+    }
+    inceptionYear = "2025"
+    vendor = "Dremio"
+    copyright = "Copyright (c) ${LocalDate.now().year} Dremio"
+  }
+
+  files {
+    subprojects.forEach { project ->
+      glob {
+        pattern.set(project.layout.buildDirectory.dir("libs").get().asFile.absolutePath + "/**.jar")
+      }
+    }
+  }
+
+  signing {
+    active.set(Active.ALWAYS)
+    verify.set(false) // requires the GPG public key to be set up
+    armored.set(true)
+  }
+
+  hooks {
+    condition.set("'{{ Env.CI }}' == true")
+    script {
+      before {
+        filter {
+          includes.set(listOf("session"))
+        }
+        run.set("""
+        echo "### {{command}}" >> ${'$'}GITHUB_STEP_SUMMARY
+        echo "| Step | Outcome |" >> ${'$'}GITHUB_STEP_SUMMARY
+        echo "| ---- | ------- |" >> ${'$'}GITHUB_STEP_SUMMARY
+        """.trimIndent())
+      }
+      success {
+        filter {
+          excludes.set(listOf("session"))
+        }
+        run.set("""
+        echo "| {{event.name}} | :white_check_mark: |" >> ${'$'}GITHUB_STEP_SUMMARY
+        """.trimIndent())
+      }
+      success {
+        filter {
+          includes.set(listOf("session"))
+        }
+        run.set("""
+        echo "" >> ${'$'}GITHUB_STEP_SUMMARY
+        """.trimIndent())
+      }
+      failure {
+        filter {
+          excludes.set(listOf("session"))
+        }
+        run.set("""
+        echo "| {{event.name}} | :x: |" >> ${'$'}GITHUB_STEP_SUMMARY
+        """.trimIndent())
+      }
+      failure {
+        filter {
+          includes.set(listOf("session"))
+        }
+        run.set("""
+        echo "" >> ${'$'}GITHUB_STEP_SUMMARY
+        echo "### Failure" >> ${'$'}GITHUB_STEP_SUMMARY
+        echo "\`\`\`" >> ${'$'}GITHUB_STEP_SUMMARY
+        echo "{{event.stacktrace}}\`\`\`" >> ${'$'}GITHUB_STEP_SUMMARY
+        echo "" >> ${'$'}GITHUB_STEP_SUMMARY
+        """.trimIndent())
+      }
+    }
+  }
+
+  release {
+    github {
+      releaseName.set("{{projectNameCapitalized}} {{projectVersionNumber}}")
+      repoOwner.set("dremio")
+      name.set("iceberg-auth-manager")
+      branch.set("main")
+      skipTag.set(true)
+      tagName.set("authmgr-{{projectVersion}}")
+      commitAuthor {
+        name.set("{{projectNameCapitalized}} Release Workflow [bot]")
+        email.set("authmgr-release-workflow-noreply@dremio.com")
+      }
+      milestone {
+        close.set(true)
+        name.set("{{projectVersionNumber}}")
+      }
+      issues {
+        enabled.set(true)
+        comment.set("🎉 This issue has been resolved in version {{projectVersionNumber}} ([Release Notes]({{releaseNotesUrl}}))")
```

```suggestion
        comment.set("🎉 These issues have been resolved in version {{projectVersionNumber}} ([Release Notes]({{releaseNotesUrl}}))")
```
(:achievement_unlocked: found a typo ;) )

---

Hm - what does JR do there in this `issues {}` block?
I assumed it adds the issues to release notes.

---

No, that's the `changelog` section below. 

See here: https://jreleaser.org/guide/latest/reference/release/github.html#_issues

This section configures JReleaser to label each issue and assign the right milestone to them.

---

TIL